bazel/common/proto_common.bzl

```diff
@@ -186,6 +186,8 @@ def _compile(
 
     if plugin_output:
         args.add(plugin_output, format = proto_lang_toolchain_info.out_replacement_format_flag)
+        if proto_lang_toolchain_info.out_replacement_format_flag.startswith("--python"):
+            args.add(plugin_output, format = "--pyi_out=%s")
```

Comment:
I'm not entirely sure the best way to do this. I would prefer to just add the `--pyi_out` to python's `proto_lang_toolchain` instantiation but the `command_line = "--python_out=%s",` can only contain one `%s` string.

This is my first time in this code base so I would appreciate any suggestions.